pom.xml

```diff
@@ -119,6 +119,11 @@
 <!--			<version>RELEASE</version>-->
 			<scope>compile</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springdoc</groupId>
+			<artifactId>springdoc-openapi-ui</artifactId>
+			<version>1.5.8</version>
```

Comment:
Please use variables for versioning

Reply:
https://github.com/ishmum123/studentlog/pull/28/commits/d774b39665d7d33cb5d17ac01b3c2793f1e5d969